songpal/device.py

```diff
@@ -316,9 +370,11 @@ async def set_bluetooth_settings(self, target: str, value: str) -> None:
         params = {"settings": [{"target": target, "value": value}]}
         return await self.services["avContent"]["setBluetoothSettings"](params)
 
-    async def get_custom_eq(self):
+    async def get_custom_eq(self, target=""):
         """Get custom EQ settings."""
-        return await self.services["audio"]["getCustomEqualizerSettings"]({})
+        return await self.services["audio"]["getCustomEqualizerSettings"](
+            {"target": target}
+        )
```

Comment:
Just a note, probably no need for action. This changes the call for everyone, but IIRC my old device did not support this and I cannot recall if someone has ever tested it, so maybe this would've been the correct call in the first place.

Reply:
Yeah, I understand that, and it could be breaking. Because there is no version change there is no way to differentiate. Based on the Sony site example it should be sent with `{"target":""}` so I thought I would make it the default.

We could try and if we get an `IllegalArgument`, then fallback to how it was?

Reply:
Another option could be to check the `method.signature.inputs`, and hopefully devices that don't work with a parameter of `"target":""` won't return it with the `getMethodTypes`.